client/src/rpc_client.rs

```diff
@@ -487,17 +482,8 @@ impl RpcClient {
                 }
                 let Response {
                     context,
-                    value: mut rpc_account,
+                    value: rpc_account,
                 } = serde_json::from_value::<Response<Option<UiAccount>>>(result_json)?;
-                if let Some(ref mut account) = rpc_account {
-                    if let Binary(_) = &account.data {
-                        let tmp = Binary64(String::new());
-                        match std::mem::replace(&mut account.data, tmp) {
-                            Binary(new_data) => account.data = Binary64(new_data),
-                            _ => panic!("should have gotten binary here."),
-                        }
-                    }
-                }
```

Comment:
The problem with `Binary(String)` and `Binary64(String)` is visible here.  Serde can't distinguish between the two on deserialization with the the `untagged` attribute, so this workaround was previously needed for Rust RPC clients.